nixos/maintainers/scripts/ec2/create-amis.sh

```diff
@@ -15,18 +15,22 @@
 # set -x
 set -euo pipefail
 
+var () { true; }
```

Comment:
This is soley to improve the semantic in a less verbose way. I could instead do something like `state_dir=${state_dir:-$HOME/amis/ec2-images}`, it just hurt my eyes a bit :sweat_smile: